doc/source/example-mxnet-walk-through.rst

```diff
@@ -0,0 +1,349 @@
+    ) -> Tuple[float, float]:
+        # Use Accuracy and Cross Entropy Loss as the evaluation metric.
+        accuracy_metric = mx.metric.Accuracy()
+        loss_metric = mx.metric.CrossEntropy()
+        metrics = mx.metric.CompositeEvalMetric()
+        for child_metric in [accuracy_metric, loss_metric]:
+            metrics.add(child_metric)
+        # Reset the validation data iterator.
+        val_data.reset()
+        # Loop over the validation data iterator.
+        for batch in val_data:
+            # Splits validation data into multiple slices along batch_axis
+            # and copy each slice into a context.
+            data = gluon.utils.split_and_load(batch.data[0], ctx_list=device, batch_axis=0)
+            # Splits validation label into multiple slices along batch_axis
+            # and copy each slice into a context.
+            label = gluon.utils.split_and_load(
+                batch.label[0], ctx_list=device, batch_axis=0
+            )
+            outputs = []
+            for x in data:
+                outputs.append(net(x).softmax())
+            # Updates internal evaluation
+            metrics.update(label, outputs)
+        return metrics.get_name_value()
+
+Having defined defining the data loading, model architecture, training, and evaluation we can put everything together and train our model on MNIST. Note that the GPU/CPU device for the training and testing is defined within the :code:`ctx` (context).  
+
+.. code-block:: python
+
+    def main():
+        # Setup context to GPU and if not available to CPU
+        DEVICE = [mx.gpu() if mx.test_utils.list_gpus() else mx.cpu()]
+        # Load train and validation data
+        train_data, val_data = load_data()
+        # Define sequential model
+        net = model()
+        init = nd.random.uniform(shape=(2, 784))
+        net(init)
+        # Start model training based on training set
+        train(net=net, train_data=train_data, epoch=5, device=DEVICE)
+        # Evaluate model using loss and accuracy
+        eval_metric = test(net=net, val_data=val_data, device=DEVICE)
+        acc = eval_metric[0]
+        loss = eval_metric[1]
+        print("Evaluation Loss: ", loss)
+        print("Evaluation Accuracy: ", acc)
+
+    if __name__ == "__main__":
+            main()
+
+You can now run your MXNet machine learning workload:
+
+.. code-block:: python
+
+    python3 mxnet_mnist.py
+
+So far this should all look fairly familiar if you've used MXNet or even PyTorch before.
+Let's take the next step and use what we've built to create a simple federated learning system consisting of one server and two clients.
+
+MXNet meets Flower
+------------------
+
+So far it was not possible to train a MXNet workload federated since it was not supported yet. However, Flower is fully agnostic to machine learning frameworks and can take any model parameters and aggregate them. This section will show you how Flower can be used to federate MXNet workloads.
```

Comment:
```suggestion
So far, it was not easily possible to use MXNet workloads for federated learning because federated learning is not supported in MXNet. Since Flower is fully agnostic towards the underlying machine learning framework, it can be used to federated arbitrary machine learning workloads. This section will show you how Flower can be used to federate our centralized MXNet workload.
```